generator/src/main/java/de/starwit/generator/services/AppSetupService.java

```diff
@@ -56,16 +53,12 @@
 	 */
 	@Transactional(propagation = Propagation.NEVER)
 	public void setupAndGenerateApp(GeneratorDto dto) throws NotificationException, EntityNotFoundException {
-		App app = appService.findAppByIdOrThrowExeption(dto.getApp().getId());
+		App app = appService.findById(dto.getApp().getId());

```

Comment:
Is there an exception thrown when not found?

Reply:
yes: return this.getRepository().findById(id).orElseThrow(() -> new EntityNotFoundException(String.valueOf(id)));